app/javascript/src/NewService/components/ServiceDiscoveryForm.jsx

```diff
@@ -9,11 +9,13 @@ import {fetchData} from 'utilities/utils'
 import {PROJECTS_PATH} from 'NewService'
 
 type Props = {
+  apiap: boolean,
```

Comment:
This won't be needed if you do the changes suggested in https://github.com/3scale/porta/pull/1119/files#diff-708bb89f617b27eb6d9826c93af38a90R30